File: fibonacci/python_fibonacci/fibonacci.py
import sys

def fibonacci(n):
  if n == 0:
    return 0
  elif n == 1:
    return 1
  else:
    return (fibonacci(n-1) + fibonacci(n-2))

def main():
  if len(sys.argv) != 1:
    print ("Usage: %s <noargs>" % sys.argv[0])
    sys.exit()
  for idx_i in range(1, 25):
    print ("fibonacci(%d)=<%d>" % (idx_i, fibonacci(idx_i)))
  sys.exit(), end

main()

File: fibonacci/ocaml_fibonacci/fibonacci.ml
(* rec - implies recursion *)
let rec fibonacci(x) = if x = 0 then 0 else if x = 1 then 1 else fibonacci(x - 2) + fibonacci(x - 1);;

let main () =
  if  Array.length Sys.argv <> 1 then
    begin
      Printf.printf "%s <noargs>\n" Sys.argv.(0);
      exit 1
    end
  else
    begin
    for idx_i = 1 to 24 do
      Printf.printf "fibonacci(%d)=<%d>\n" idx_i (fibonacci(idx_i))
    done;
    end

let _ = main ()
